plugins/history/chathistoryhandler.cpp

```diff
@@ -11,7 +14,11 @@ K_EXPORT_PLUGIN(HistoryPluginFactory( "kopete_history" ))
 ChatHistoryHandler::ChatHistoryHandler(QObject *parent, const QVariantList &)
 	: Kopete::Plugin(HistoryPluginFactory::componentData(), parent)
```

Comment:
Why don't we name this as `ChatHistoryPlugin` instead of `ChatHistoryHandler`. what do you think?